homeworks/nikita.karchinskiy_Nikkarchinskiy/4-moduls/index.js

```diff
@@ -0,0 +1,7 @@
+// import { daysInMonthFnc, mondaysInMonth } from './exports.js';
```

Comment:
Your code is commented on, so it would not work. You need to move all DOM logic here and make it work